packages/css/src/components/form-error-list/README.md

```diff
@@ -0,0 +1,35 @@
+<!-- @license CC0-1.0 -->
+
+# Form Error List
+
+Use this component at the top of a page to summarise any errors a user has made.
+When a user makes an error, you must show both a Form Error List and an Error Message above each answer that contains an error.
+
+## Guidelines
+
+- Always show a Form Error List when there is a validation error, even if there’s only one.
```

Comment:
You've answered my previous comment (lol). But I disagree. Imagine you have an email field for a newsletter in a footer and you make a typo like miss the `@`, showing this component and a formfield error would seem like overkill and possible destroy the footer layout.

Reply:
That's a pretty good reason to not have an e-mail field in the footer imo 😁 I think we removed that from the design as well.

This validation pattern has been pretty thoroughly tested, by Adam Silver / Gov.uk, NLDS, the Danish Design system etc. Simply showing an error message above a field is not obvious enough, so we should avoid it.